pkg/report/writer.go

```diff
@@ -233,3 +235,34 @@ func (tw TemplateWriter) Write(results Results) error {
 	}
 	return nil
 }
+
+func toSarifRuleName(vulnerabilityType string) string {
+	var ruleName string
+	switch vulnerabilityType {
+	case vulnerability.Ubuntu, vulnerability.Alpine, vulnerability.RedHat, vulnerability.RedHatOVAL,
+		vulnerability.Debian, vulnerability.DebianOVAL, vulnerability.Fedora, vulnerability.Amazon,
+		vulnerability.OracleOVAL, vulnerability.SuseCVRF, vulnerability.OpenSuseCVRF, vulnerability.Photon,
+		vulnerability.CentOS:
+		ruleName = "Os Package Vulnerability "
+	case "npm", "yarn", "nuget", "pipenv", "poetry", "bundler", "cargo", "composer":
+		ruleName = "Programming Language Vulnerability "
+	default:
+		ruleName = "Other Vulnerability "
+	}
+	return ruleName + strings.Title(vulnerabilityType)
+}
+
+func toSarifErrorLevel(severity string) string {
+	var sarifErrorLevel string
+	switch severity {
+	case "CRITICAL", "HIGH":
+		sarifErrorLevel = "error"
+	case "MEDIUM":
+		sarifErrorLevel = "warning"
+	case "LOW", "Unknown":
```

Comment:
```suggestion
	case "LOW", "UNKNOWN":
```

See https://github.com/aquasecurity/trivy-db/blob/68e2a005ae7900d27c14738599541c8e082d4bc3/pkg/types/types.go#L38 for all the possible severity levels.

Reply:
Fixed